packages/sdk-react/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@fuels/sdk-react",
+  "version": "0.0.9",
+  "license": "Apache-2.0",
+  "main": "src/index.ts",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.js",
+    "module": "dist/index.mjs",
+    "types": "dist/index.d.ts",
+    "typings": "dist/index.d.ts",
+    "exports": {
+      ".": {
+        "require": "dist/index.js",
+        "default": "dist/index.mjs"
+      }
+    },
+    "files": [
+      "dist"
+    ]
+  },
+  "scripts": {
+    "build": "pnpm ts:check && tsup --dts",
+    "ts:check": "pnpm tsc --noEmit"
+  },
+  "peerDependencies": {
+    "fuels": ">=0.54.0",
```

Comment:
We should turn react into a peerDependencie to avoid conflicts with the one been used by the user